test/operators.jl

```diff
@@ -49,7 +49,7 @@ p = 1=>:foo
 @test xor(2) == 2
 @test (⊻)(2) == 2
 
-# @test ctranspose('a') == 'a' # (c)transpose of Chars no longer supported
+# @test adjoint('a') == 'a' # (c)transpose of Chars no longer supported
```

Comment:
Hmm... I think we can delete this now.

(I have an upcoming PR which will let us transpose arrays of `Char`s and `String`s and so-on, so the context of this whole line will be rather out-of-date.)